pokemongo_bot/cell_workers/move_to_fort.py

```diff
@@ -109,7 +109,7 @@ def work(self):
                     formatted='Arrived at fort.'
                 )
 
-        return WorkerResult.SUCCESS
```

Comment:
Because you do not want another moving tasks to dictate the position. You are the one deciding at that point.